### DISCUSSION TITLE
Github link for Emma Dawson's github link is wrong (small issue)

### DISCUSSION THREAD

**Discussion by dinxsh** [EddieHubCommunity/support#4746] [First-Timers] [answered]
# Wrong Github Link on official website 
https://www.eddiehub.org/ambassador-program
![Screenshot 2022-07-27 144121](https://user-images.githubusercontent.com/90450035/181210022-6999e7d8-37c3-4350-841c-a034fbdce3ac.png)

The github link above is  wrong and leads to - 

![Screenshot 2022-07-27 144444](https://user-images.githubusercontent.com/90450035/181210376-d50aa526-c3d1-4a0b-be9d-20b9bbd47a09.png)

**Comment by dinxsh** [comment, marked as answer]
# Solution ⚙

can be solved by changing link to href="https://github.com/EmmaDawsonDev" 
(correct link)

**Comment by kartikgajjar7** [comment]
can you provide link of issue?
